src/@chakra-ui/gatsby-plugin/components/index.ts

```diff
@@ -1,3 +1,9 @@
-export * from "./Link"
-export * from "./Button"
-export * from "./Tag"
+import { Button } from "./Button"
+import { Link } from "./Link"
+import { Tag } from "./Tag"
+
+export default {
+  Button,
+  Link,
+  Tag,
+}
```

Comment:
This took me a very long time to detect 😆 hate these kind of issues.

Once I removed the `.babelrc` for some reason these exports stop working as I was expecting. I had to export these in a more manual way, with the expected structure.